src/HasTags.php

```diff
@@ -310,4 +314,17 @@ protected function syncTagIds($ids, string $type = null, $detaching = true)
             $this->tags()->touchIfTouching();
         }
     }
+
+    /**
+     * Check the query "from" statement for an alias and return it if found
+     *
+     * @param  \Illuminate\Database\Eloquent\Builder $query
+     * @return string
+     */
+    protected function convertForQueryAlias(Builder $query): string
```

Comment:
This function returns the alias right?

Rename this to `getTablesAlias`, might be a bit more clear